src/routes/sections/withdraw-section.svelte

```diff
@@ -172,6 +178,10 @@
       nativeTokensToSend,
       formInput.nftIDToSend,
     );
+
+    if (!isWithdrawing) {
```

Comment:
isnt there a way to know if it was succesful ot not? resetting if there was a fail is bad UX because the user has to write everything again. We should reset only onSuccess imo